project_event/models/project_task.py

```diff
@@ -401,46 +401,44 @@ def create_children_from_activity_create(self, children, parent_id):
 
     @api.multi
     def write(self, vals):
-        if self.is_activity():
-            return self.write_activity(vals)
-        else:
-            self.update_reservation_event(vals)
-            return super(Task, self).write(vals)
+        super(Task, self).write(vals)
```

Comment:
You should return super(Task, self).write(vals)

Reply:
what is the implication?